src/core/utilities/aria-directives.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Directive,
+  ElementRef,
+  Input,
+  Renderer,
+} from 'angular2/core';
+import {AttributeTargetDirectiveInterface, MdAttributeTarget} from './attribute-target';
+import {BooleanFieldValue} from '../annotations/field-value';
+
+
+/**
+ * aria directive, handling aria parts of the standard.
+ */
+@Directive({
+  selector: '[aria-label], [aria-labelledby], [aria-disabled], [aria-required], [aria-invalid]'
+})
+export class MdAria implements AttributeTargetDirectiveInterface {
```

Comment:
If we keep this (wrt concerns about performance), the name is too nonspecific. It would be something like `MdAriaAttributeForwarder`